Jenkinsfile

```diff
@@ -0,0 +1,16 @@
+@Library('linkurious-shared')_
+
+nodeJob {
+  // General
+  projectName = "linkurious/lke-plugin-csv-importer"
+  runUnitTests = false
+  runE2eTests = false
+
+  createGitTag = true
+
+  //static asset upload
+  runPrivateNpmPublish = false
+  binaries = ["lke-plugin-data-table.lke"]
```

Comment:
@aminerhanemi it should be fixed to the csv import

Reply:
and basicaly match the 'projectName'